lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -16139,4 +16139,13 @@ msgid "At least one column must be selected"
 msgstr "Au moins une colonne doit être sélectionnée"
 
 msgid "The duration must be lesser than a year"
-msgstr "La durée doit être inférieure à une année"
\ No newline at end of file
+msgstr "La durée doit être inférieure à une année"
+
+#  msgid "Current"
+#  msgstr "Taille courante"
+
+#  msgid "Medium"
+#  msgstr "Taille moyenne"
+
+#  msgid "Small"
+#  msgstr "Taille petite"
```

Comment:
```suggestion
msgid "Current"
msgstr "Courante"

msgid "Medium"
msgstr "Moyenne"

msgid "Small"
msgstr "Petite"
```

Reply:
Or change `Current` to `Current size`, etc...

Reply:
Hi, ok i'll push this change